engine-tests/src/tests/eth_connector.rs

```diff
@@ -98,12 +98,10 @@ fn init_contract(
     contract_account
 }
 
-fn validate_eth_address(address: &str) -> EthAddress {
+fn validate_eth_address(address: &str) -> Address {
     let data = hex::decode(address).unwrap();
     assert_eq!(data.len(), 20);
-    let mut result = [0u8; 20];
-    result.copy_from_slice(&data);
-    result
+    Address::from_slice(&data)
```

Comment:
Could use `Address::decode(address).unwrap()` instead and this function would be 1 line instead of 3.